src/views/list/ListPrincipalView.vue

```diff
@@ -42,7 +42,7 @@
 
     <section class="weight-1 h-full bg-black-100">
       <await
-        name="listPrincipal"
+        name="hardQuery"
```

Comment:
I am not a big fan of hardQuery and softQuery names, but I don't mind. And I don't think we need to have a discussion about it, let's just keep this names for now